src/openbiolink/edge.py

```diff
@@ -2,12 +2,13 @@
 
 
 class Edge:
-    def __init__(self, id1: str, type : EdgeType, id2: str, source: "", qScore = None):
+    def __init__(self, id1: str, type : EdgeType, id2: str, source: "", qScore = None, sourcedb =None):
```

Comment:
I added a new parameter to the edge class called sourcedb. I noticed that there is already a parameter called source, i couldn't find where it is used, but didn't want to override it.